README.md

```diff
@@ -38,7 +38,7 @@ discord.js is a powerful [Node.js](https://nodejs.org) module that allows you to
 - 100% coverage of the Discord API
 
 ## Installation
-**Node.js 10.0.0 or newer is required.**  
+**Node.js 10.2.0 or newer is required.**  
```

Comment:
This shouldn't change until an official announcement is made about a version bump

Reply:
Thanks for the feedback! :) 

I'll wait for confirmation from @iCrawl , however this change does rely on v10.2 because of the use of `refresh()`. Let me know the best way to approach things if we can't bump the version (maybe even a note to say this feature requires v10.2?).

Cheers.